.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -41,9 +41,11 @@ body:
         - Windows 8
         - Windows 7
         - Windows Other
-        - Mac OS
+        - macOS (Intel)
+        - macOS (Apple Silicon)
         - Linux x86
         - Linux x86_64
+        - Linux ppc64le
```

Comment:
Well, PPC builds are not officially supported, so I'm unsure about this. But I'll defer to @kcgen who is the only person here who cares about those builds 😄